src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Header comments should be provided, else developers adding overriding methods will have no idea about the expected behavior.